IBM/carbon-ui-builder
# Implement actions for button in react export

## Related Tickets & Documents



- Related Issue #
- Closes #


## What type of PR is this? (check all that apply)






## Scope


This PR adds the features of changing visibility and disabled property of a button using another button in the react export code

## Implementation



Parse the `actions` array in the JSON into setState statements in the react export code. Only works on button signals and slots for now.

## Screenshots/Recordings/Diagrams:



N/A

## How to test


Create a fragment with 2 or more buttons, add action interactions that controls disabled or visibility, observe the setState statements in the react export code. Optionally, run the export code in a code sandbox.
